std-bits/tableau/src/main/java/org/enso/tableau/HyperReader.java

```diff
@@ -88,7 +91,7 @@ private static HyperProcess getProcess() throws IOException {
     if (process == null || !process.isOpen()) {
       var contextClassLoader = Thread.currentThread().getContextClassLoader();
       try {
-        Thread.currentThread().setContextClassLoader(HyperReader.class.getClassLoader());
+        Thread.currentThread().setContextClassLoader(new TableauClassLoader());
```

Comment:
There seems to be:
```
~/bin/graalvm/bin/javap -cp built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/lib/Standard/Tableau/0.0.0-dev/polyglot/java/tableauhyperapi.jar com.tableau.hyperapi.impl.SharedLibraryProvider
Compiled from "SharedLibraryProvider.java"
public interface com.tableau.hyperapi.impl.SharedLibraryProvider {
  public abstract java.lang.String getSharedLibraryBaseName();
}
```
cannot it be used for loading the JNI library without these classloading tricks?

Reply:
Yes, I saw that. Sadly I couldn't make it to work that way; JNA gets in the way.

Reply:
OK, thanks.